integration/presto/pom.xml

```diff
@@ -375,9 +374,9 @@
       <version>2.5</version>
```

Comment:
Shouldn't it be commons-lang3 version 3.5? In other places it depends on that

Reply:
the group id is not `org.apache.commons` This is `commons-lang`
Not sure why this dependency needed. It is from base code. Let me check

Reply:
removed now